frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -167,6 +181,29 @@ const getTopologyEdgeItems = (dc: K8sResourceKind, resources: K8sResourceKind[])
       });
     }
   });
+
+  _.forEach(edgesFromServiceBinding(dc, serviceBindingRequests), (edge) => {
+    // handles multiple edges
+    const targetNode = _.get(
+      _.find(resources, (deployment) => {
+        const targetFound =
+          _.get(deployment, 'metadata.ownerReferences[0].name') === edge.resourceRef &&
+          _.get(deployment, 'metadata.ownerReferences[0].kind') === edge.kind;
+        return targetFound;
+      }),
+      ['metadata', 'uid'],
+    );
+    const uid = _.get(dc, ['metadata', 'uid']);
+    if (targetNode) {
+      edges.push({
```

Comment:
Save the trouble of lookup when you delete the edge by storing the SBR in the data prop of the edge.

Reply:
Yes, initially I also thought of doing that but I was not sure if it makes sense to store the SBR in the edge data itself.

Reply:
Done.